sdk/src/types/block/semantic.rs

```diff
@@ -17,91 +17,110 @@ use crate::types::block::{
 
 /// Errors related to ledger types.
 #[derive(Debug)]
-pub enum ConflictError {
-    /// Invalid conflict byte.
-    InvalidConflict(u8),
+pub enum TransactionFailureError {
+    /// Invalid transaction failure reason byte.
+    InvalidReason(u8),
```

Comment:
If there is really only one variant after all this time, I think I'd just add a variant to the main error type...

Reply:
We sure it's not somewhere we temporarily commented ? If really not used, I'd remove yeah

Reply:
Like, how is it in JSON? As an integer? If yes, surely it needs to use this?

Reply:
Well actually we just never used the conflict reason and in the response it was just an u8

Reply:
So yeah we still need it

Reply:
Can it be added to the main error then?

Reply:
Yes, changed